libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/location/NavigationLocationProvider.kt

```diff
@@ -122,6 +140,7 @@ class NavigationLocationProvider : LocationProvider {
      * @see LocationObserver
      * @see MapboxNavigation.registerLocationObserver
      */
+    @CallSuper
```

Comment:
Thinking about making this final.. Leaving it open provides more flexibility but it's not covering a real use case.

Once it is open it cannot be closed, so am debating making it `final`. Maybe someone would want to intercept these events 🤔